packages/wallet-frontend/src/lib/api/user.ts

```diff
@@ -135,7 +135,7 @@ const createUserService = (): UserService => ({
   async signUp(args): Promise<SignUpResponse> {
     try {
       const response = await httpClient
-        .post('signup', {
+        .post('auth/signup', {
```

Comment:
```suggestion
        .post('signup', {
```

Reply:
moved backend-update to /signup /login for now aswell, without the /auth